pants.toml

```diff
@@ -16,6 +16,11 @@ backend_packages = [
   "pants.backend.experimental.scala.lint.scalafmt",
 ]
 
+# Opt in to future behaviors that will become the default in Pants v2.14.
+# These directives should be removed in Pants v2.14.x and higher.
+use_deprecated_directory_cli_args_semantics = false
+use_deprecated_pex_binary_run_semantics = false
```

Comment:
This shouldn't be necessary since this repo doesn't use the Python backend... are you seeing otherwise?

Reply:
Yes the message appeared when I ran `./pants package ::`.